client/src/components/MovieShow.js

```diff
@@ -25,7 +28,12 @@ const MovieShow = (props) => {
     getMovie();
   }, []);
 
-  return <h1>{movie.title}</h1>;
+  return (
+    <Fragment>
```

Comment:
use an empty fragment `<>` that we have available due to babel, meaning we also do not need to import it at the top